modules/ROOT/pages/install-multiple-instances.adoc

```diff
@@ -2,14 +2,12 @@
 
 Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.
 
-[NOTE]
-To use this feature, you must upgrade to the minimum version of 2.2.5 of the Runtime Fabric agent that supports multiple instances.
-
-[NOTE]
-There will be 1:1 mapping from the agent namespace to the app namespace. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.
+* To use this feature, you must upgrade to the minimum version of 2.2.5 of the Runtime Fabric agent that supports multiple instances.
+* There will be 1:1 mapping from the agent namespace to the app namespace. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.
```

Comment:
Avoid future tense and "There is" construction. 

Agent namespaces map one-to-one to app namespaces. 
or 
For each agent namespace, you can have one app namespace.